index.html

```diff
@@ -947,13 +945,13 @@ <h3>Implicit release of pointer capture</h3>
         </section>
     </section>
     <section>
-        <h1><dfn>Coalesced and predicted events</dfn></h1>
+        <h1><dfn class="export">Coalesced and predicted events</dfn></h1>
```

Comment:
I wonder if this needs to have export. Feels very PointerEvents spec internal thing.